weblate/settings_example.py

```diff
@@ -677,6 +677,9 @@
 # Use simple language codes for default language/country combinations
 SIMPLIFY_LANGUAGES = True
 
+# Default number of elements when pagination is active
+DEFAULT_PAGE_LIMIT = 100
```

Comment:
Please move this to weblate/trans/models/_conf.py which contains default values (so that it doesn't break when somebody doesn't update the settings to match new example).

Reply:
Do you want me to append 
`.. versionadded:: 4.7.1 `
to rst admin doc?